### DISCUSSION TITLE
Production sites

### DISCUSSION THREAD

**Discussion by EvanEdwards** [php/frankenphp#244] [General]
I noticed that the "NOT FOR PRODUCTION USE" warning banners seem to have been removed from public facing sites.

At this point is there anything keeping FrankenPHP from being used in at minimum a trivial production manner?  I.e., public server for sites that people aren't depending on?  Or depending on, but for purposes that can easily tolerate downtime and a reversion back to classic caddy+PHP if needed?

I'm mostly wondering about security issues or things that wouldn't show up in unit tests of our PHP code.  In my own case, I've happily been using a static build on my own dev setup at this point, and am considering pushing it to the other devs and moving a few hobby/placeholder sites over on "production" as a test toward a full switchover in the future as we move from nginx to requiring Mercure.  I have agency to do so, but a sysadmin who is dubious (the best ones are paranoid, so I'm okay with "dubious").

Or have people already started running their organization or business on it and are happily serving pages and apps with no problem other than having to track the project a little closer due to the rapid pace of development?  (I myself missed a couple months due to a family tragedy, so apologies if this has been announced: I did look to see if I could find anything).

Finally, I am fully aware any technical choice is a judgement call.  I started this as a discussion (as it isn't an issue with a resolution) to hear perspectives and am not expecting absolute answers.  But please feel free to express strong opinions!

**Comment by withinboredom** [comment]
I'm running a couple of smaller (hobby) services in production -- getting a decent-ish amount of traffic. At work, we were discussing moving a subset of microservices over to Frankenphp in the near future. Our PHP pods are quite complex and would benefit greatly from switching. This probably won't happen until later this quarter, but it is happening.

**Comment by EvanEdwards** [comment]
(Many Months Later)...

## Just in case this pops up in a web search, FrankenPHP has been production ready and has been in use by many people, including my company and my own personal sites, for awhile now.  It is tested in production and ready for you to use.
